### DISCUSSION TITLE
DreamBerd: Release Notes: Community Update

### DISCUSSION THREAD

**Discussion by TodePond** [TodePond/GulfOfMexico#242] [Announcements]
It's time for another DreamBerd release! We've got loads of juicy new features for you to get stuck into. Hop over to the [examples page](https://github.com/TodePond/DreamBerd/blob/main/examples/examples/Examples.md) to see them in action. Or read below for the full details 👇

But first, let's do a community update.

## Community Update

DreamBerd creator, Lu Wilson, delivered a **tech talk** about the project. It happened at the latest Future of Code meetup here in London. 

[Go watch it now](https://youtu.be/52vmjZnxJb8)!

Now, onto the release notes.

## Release Notes

Let's dive in to all the new features.

But just before we do that, we have a small community update for you.

### Community Update

Popular streamer TheDreamBerdeagen did an educational read-through of the DreamBerd language.

[Go watch it now](https://youtu.be/tDexugp8EmM)!

After that, come back here for the release notes 👇

### Release Notes

Let's take a look at this week's release notes. As always, let us know what you think about these changes. Find us on [GitHub](https://github.com/TodePond/DreamBerd)!

But just before we do that, here's a quick community update about the state of the project.

## Community Update

We're very excited to tell you that DreamBerd has now secured a sponsor! This means that future development of the project is now guaranteed.

To all the open-source contributors who helped DreamBerd get to this milestone: Please note that you will *not* receive your share of the compensation.

Though, to be clear, the sponsor is compensating us with **exposure**, not money. So you might not be interested anyway.

More info on our sponsor below 👇

But now, let's do the release notes.

# Release Notes

Welcome to this week's release notes. Our engineers have been hard at work. And we've got plenty of new features for you to explore.

But before that, we've got a small community update for you.

## Community Update

DreamBerd creator, Lu Wilson, was recently interviewed on TheDreamBerdeagen's stream.

You can watch the full interview...

You can watch...

*sorry, just a second*

...

*ok thank you for waiting*

Let's move on to the release notes now.

## Release Notes

There are lots of exciting new features in this week's DreamBerd release.

But first, here's a community update from our sponsor.

## Community Update (sponsored)

*Here at the DreamBerd Foundation, we love programmers. There's not a more compassionate - nor a more thoughtful, kind - group of people than programmers.*

*We **love** programmers, almost as much as you love one another.*

*So we're extremely pleased to announce that we are now the official sponsor of DreamBerd.*

*That's right! The DreamBerd Foundation is sponsoring DreamBerd. We're really excited about the possibilities that this partnership will provide. In the words of ChatGPT, "happy coding"! 💪*

# Release Notes

Let's do the release notes. We're almost there.

There's just one more community update to go, and then it's time. Keep reading 👇

# Community Update

The DreamBerd Foundation has now started work on an ECMAScript library called [Airspace](https://github.com/TodePond/Airspace).

We are looking for early contributors to get an implementation going as soon as possible.

Now... We're finally at the release notes.

# `Release Notes`

Release notes. Here they are. You've made it. Except, there's just a. There's a community. Community. Community update.

# `Community Update`

Berd.

Berd.

Berd!

[The video](https://youtu.be/8TFuKwdixlU), berd.

BERD.

> Sorry to interrupt. I'm writing the release notes right now, and I've just realised that they're not actually release notes. Instead, they're community updates. A real example of release notes... is this:
>
> ## Release Notes
> Welcome back to another weekly DreamBerd release. We're hard at work on some behind-the-scenes things right now. But we've still got some shiny new things for you to explore.
>
> But first, we have a quick community update for you.

#### Community Update
Unfortunately, our sponsor has recently cancelled their partnership with us. If you would like to help to keep the project alive then you can.

Simply become a DreamBerd patron by paying 5 dollars a month to the [Stonewall charity](https://www.stonewall.org.uk/). This will help to prevent the DreamBerd creator from losing their human rights, thus ensuring the future of the project.

You can find the full release notes in our next release notes.

<hr /><em>This discussion was created from the release <a href='https://github.com/TodePond/DreamBerd/releases/tag/v.release.notes'>DreamBerd Release Notes: Community Update</a>.</em>

**Comment by tiago-macedo** [comment]
I understand that the release notes' main purpose is to present the latest features and fixes, but I'd like to know more about what's going on in the community.

**Comment by ari-party** [comment]
To the moon.
